tests/testsuite.at

```diff
@@ -246,6 +246,7 @@ m4_ifblank(m4_join([],ENV_CATEGORIES), [
 m4_foreach(current, [CATEGORIES],
 [
   AT_BANNER([current])
+  m4_if($SWIG_TRUE, )
```

Comment:
This should check if `$SWIG_FALSE` is the string `#`. If so, then `$SWIG_TRUE` will be the empty string here, and the swig tests should be enabled. Otherwise, if `$SWIG_FALSE` is the empty string, then `$SWIG_TRUE` will be the string `#` and the swig tests should not be enabled.